src/main/python/main.py

```diff
@@ -0,0 +1,92 @@
+from fbs_runtime.application_context.PyQt5 import ApplicationContext
+from fbs_runtime.application_context import cached_property
+from os import environ
+
+from mainapp import MainApp
+
+# Removed by sgomena on 1/29/19 after directory structure refactoring
+# sys.path.append(path.join(path.dirname(__file__), '..'))
+
+
+class AppContext(ApplicationContext):
+
+    def __init__(self, *args, **kwargs):
+        super(AppContext, self).__init__(*args, **kwargs)
+        self.mainapp = MainApp(self)
+
+    @cached_property
+    def dashboard_style(self):
+        return self.load_style("dashboard.qss")
+
+    @cached_property
+    def app_style(self):
+        return self.load_style("app.qss")
+
+    @cached_property
+    def login_style(self):
+        return self.load_style("login.qss")
+
+    @cached_property
+    def credit_history_style(self):
+        return self.load_style("credit_history.qss")
+
+    @cached_property
+    def notification_style(self):
+        return self.load_style("notification.qss")
+
+    @cached_property
+    def add_view_style(self):
+        return self.load_style("add_view.qss")
+
+    @cached_property
+    def controller_style(self):
+        return self.load_style("controller.qss")
+    
+    @cached_property
+    def popup_style(self):
+        return self.load_style("popup.qss")
+
+    @cached_property
+    def jobs_style(self):
+        return self.load_style("jobs.qss")
+
+    @cached_property
+    def settings_style(self):
+        return self.load_style("settings.qss")
+
+    @cached_property
+    def question_style(self):
+        return self.load_style("question.qss")
+
+    # def __init__(self):
+    #     popup_style = self.get_resource("popup.qss")
```

Comment:
should we just nuke this code? commented out just looks kind of ugly

Reply:
Yah, good call. I removed it.